src/libYARP_dev/include/yarp/dev/MapGrid2D.h

```diff
@@ -30,6 +30,22 @@ namespace yarp
             typedef yarp::sig::PixelMono CellData;
             typedef yarp::math::Vec2D<int> XYCell;
             typedef yarp::math::Vec2D<double> XYWorld;
+
+            struct MapOrigin
+            {
+                double x;     // in meters
+                double y;     // in meters
+                double theta; // radians
+            }; 
+
+            struct MapInfo
+            {
+                double    m_resolution;   // m/pixel
+                MapOrigin m_origin;       // pose of the map frame w.r.t. the bottom left corner of the map image
```

Comment:
If we want to visualize this messages in Doxygen-generated docs, we can use the style documented in http://www.doxygen.nl/manual/docblocks.html#memberdoc .